tests/components/erosion_deposition/test_ero_dep_mass_conservation.py

```diff
@@ -0,0 +1,122 @@
+import pytest
```

Comment:
So great to have mass balance tests! As noted in my PR comment, I did some tweaks to this.